bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```
@@ -459,7 +459,7 @@ config.refreshInterval.description = The refresh interval to poll Netatmo API (i
 
 conf-error-no-client-id = Cannot connect to Netatmo bridge as no client id is available in the configuration
 conf-error-no-client-secret = Cannot connect to Netatmo bridge as no client secret is available in the configuration
-conf-error-grant-needed = Configuration incomplete, please grant the binding to Netatmo Connect.
+conf-error-grant-needed = Complete the configuration by granting the binding to Netatmo Connect : `{0}`.
```

```suggestion
conf-error-grant-needed = Complete the configuration by granting the binding to Netatmo Connect: ''{0}''.
```

Three things:
- Removed space before colon.
- Replaced **`** by **'** which seems more usual, WDYT?
- At least when using **'**, the placeholder has to be escaped (see https://github.com/openhab/openhab-addons/issues/15460#issuecomment-1686928930)